### ISSUE TITLE
In Gujarati language,  certain information is displayed in English as well.

### ISSUE THREAD

**Issue by nehabadiani** [covid19india/covid19india.github.io#2440]
**Describe the bug**
On selecting a different language to read the page in, I selected Gujarati and on scrolling down, I noticed that quite an amount of information is still displayed in English, this could lead to incomplete information to someone viewing it in Gujarati.

**To Reproduce**

1. Go to the Navigation Bar on the left and at the bottom
2. Click on 'English' or the language you are using to view the site
3. Select 'Gujarati' from the given languages
4. Click on any state and scroll down to see certain information displayed in English instead of Gujarati

**Expected behavior**
![Screenshot (156)](https://user-images.githubusercontent.com/62693931/116187477-3c39cb00-a743-11eb-9314-a7f793f7be27.png)
![Screenshot (157)](https://user-images.githubusercontent.com/62693931/116187484-3e038e80-a743-11eb-9a5e-1c228b3e4aff.png)

As can be seen in the above screenshots, the information is displayed in English instead of Gujarati.

**Screenshots**

**Desktop (please complete the following information):**

- OS: Windows
- Browser: Chrome
- Version: 90.0.4430.85

**Smartphone (please complete the following information):**

- Device: iPhone XR
- OS: 14.4
- Browser Safari
- Version 14

**Additional context**

**Comment by Ru28** [comment]
yes, @nehabadiani , you are right. Some column header is in English language.
